src/js/serial_backend.js

```diff
@@ -556,7 +556,7 @@ function onConnect() {
 }
 
 function onClosed(result) {
-    if (result) { // All went as expected
+    if (result || CONFIGURATOR.virtualMode) { // All went as expected
```

Comment:
Swallowing a falsey result here is a bad idea - we should rather fix this and return a truthy result here if we were in virtual mode.

Reply:
Fixed !